PBblas/internal/FromR8Set.ecl

```diff
@@ -0,0 +1,43 @@
+/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2016 HPCC Systems�.  All rights reserved.
+############################################################################## */
+IMPORT $.^ as PBblas;
+IMPORT PBblas.Types;
+dimension_t := Types.dimension_t;
+value_t := Types.value_t;
+Layout_Cell := Types.Layout_Cell;
+/**
+  * Take a dense matrix or submatrix and convert it to a sparse matrix in
+  * Layout_Cell format.
+  *
+  * @param mat_data   A set of REAL8 values representing a matrix in column
+  *                   first order [r1c1, r1c2, ... r1cM, r2c1, r2c2, ... rNcM]
```

Comment:
This comment describes row major (the normal approach used in C).  Column major (the approach used in Fortran) is [r1c1, r2c1, r3c1, ..., rNc1, r1c2, ..., rNcM]